app/controllers/predicates/AuthorisedAction.scala

```diff
@@ -41,7 +42,7 @@ class AuthorisedAction @Inject()()(implicit val authConnector: AuthConnector,
 
   def async(block: User[AnyContent] => Future[Result]): Action[AnyContent] = defaultActionBuilder.async { implicit request =>
 
-    implicit lazy val headerCarrier: HeaderCarrier = HeaderCarrierConverter.fromHeadersAndSession(request.headers)
+    implicit lazy val headerCarrier: HeaderCarrier = HeaderCarrierConverter.fromRequestAndSession(request, request.session)
```

Comment:
I believe journey tests will fail with this. Change it to HeaderCarrierConverter.fromRequestAndSession(request)

Reply:
HeaderCarrierConverter.fromRequestAndSession(request) has two parameters

Reply:
Sorry, I meant HeaderCarrierConverter.fromRequest(request)

Reply:
done